test/e2e/resources/10-pipeline.yaml

```diff
@@ -0,0 +1,57 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  labels:
+    name: task-s2i-go
+  name: task-s2i-go
+spec:
+  params:
+    - name: PATH_CONTEXT
+      description: The location of the path to run s2i from.
+      default: .
+      type: string
+    - name: IMAGE
+      description: Full qualified registry url, including image name and tag.
+      type: string
+    - name: TLSVERIFY
+      type: string
+      description: |
+        Sets the TLS verification flags
+    - name: VERBOSE
+      type: string
+      description: Verbose logging
+
+  workspaces:
+    - name: source
+      
+  tasks:
+    - name: git
+      taskRef:
+        name: git
+      workspaces:
+        - name: output
+          workspace: source
+      params:
+        - name: URL
+          value: https://github.com/Senjuti256/goHelloWorld
```

Comment:
Could we use a common sample, as in any s2i official Golang sample repository would be great for this use-case.

Reply:
I am looking for some sample repository and updating it.

Reply:
I have updated with a sample repository.